GENETIS_doc/jbook/index.md

```diff
@@ -1,3 +1,8 @@
 # GENETIS Home
 
-Welcome!
\ No newline at end of file
+Welcome to the GENETIS project!
+
+This documentation follows the
+[Diátaxis documentation guideline](https://diataxis.fr/)
+and is based on the dissertation by Dr. Julie Rolla, available
+[here](https://etd.ohiolink.edu/apexprod/rws_olink/r/1501/10?clear=10&p10_accession_num=osu163668789451345>).
```

Comment:
```suggestion
and is based on the [dissertation by Dr. Julie Rolla](
https://etd.ohiolink.edu/apexprod/rws_olink/r/1501/10?clear=10&p10_accession_num=osu163668789451345).
```
The text of a link should be descriptive of what it's being linked, so "here" should be avoid whenever possible.